src/man/authselect.8.adoc

```diff
@@ -118,6 +145,10 @@ To print help for the selected command run *authselect COMMAND --help*.
         be stored at {AUTHSELECT_BACKUP_DIR}/NAME. Current time with unique
         string is used as a name if no value is provided.
 
+    *--quiet, -q*:::
```

Comment:
Please update the title too
`*enable-feature* feature [-b] [--backup=NAME]::`
of --quiet or change it to something generic like 
`*enable-feature* feature [options]::`
please update the also
`*test* profile_id [features]::`